src/encoded/static/components/objectutils.js

```diff
@@ -1079,3 +1080,29 @@ Checkbox.propTypes = {
 Checkbox.defaultProps = {
     css: '',
 };
+
+
+/**
+ * Examines the `examined_loci` property, if any, of the given dataset and formats it into
+ * displayable text.
+ * @param {object} experiment Dataset containing examined_loci to display
+ * @param {boolean} includeMethod True to include expression method following the expression data
+ * @returns {string} Formatted examined_loci display
+ */
+export function computeExaminedLoci(experiment, includeMethod) {
+    const examinedLoci = [];
+    if (experiment.examined_loci && experiment.examined_loci.length > 0) {
```

Comment:
`computeExaminedLoci` need cannot trust its arguments as the function can be called by anyone rather than a fixed group in a class. So I think it would be a good idea to validate arguments before using them.